src/common/persistence/timescaledb/migrations/1680617570753-AddDailyAndWeeklyViews.ts

```diff
@@ -0,0 +1,56 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddDailyAndWeeklyViews1680617570753 implements MigrationInterface {
+  name = 'AddDailyAndWeeklyViews1680617570753';
+  transaction = false;
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE MATERIALIZED VIEW "sum_weekly" WITH (timescaledb.continuous) AS 
+    SELECT
+      time_bucket('7 day', timestamp) AS time, series, key,
+      last(value, timestamp) AS last,sum(value) AS sum
```

Comment:
when you remove `last` from queries, don't forget to also remove them from migrations

Reply:
I'll do this later on, with the next prs